setup.cfg

```diff
@@ -40,6 +40,3 @@ idna2008 =
 [options.package_data]
 * =
     LICENSE
-
-[bdist_wheel]
```

Comment:
Please revert this

Reply:
Will do, although the presence of `universal = 1` means the wheel claims compatibility with Python 2.

In the wheel filename:

* `rfc3986-2.0.0-py3-none-any.whl`
* `rfc3986-2.0.0-py2.py3-none-any.whl`

And in the metadata:

<img width="1348" alt="image" src="https://user-images.githubusercontent.com/1324225/218398347-7414df02-00e5-4afb-8107-507be139279b.png">

Claims compatibility with Python 2 in the `WHEEL` metadata: `Tag: py2-none-any`:

<img width="1353" alt="image" src="https://user-images.githubusercontent.com/1324225/218398416-7eee1e01-d2ea-4477-8e9e-31c1100ddc68.png">

Only a hash difference in `RECORD`:

<img width="1353" alt="image" src="https://user-images.githubusercontent.com/1324225/218398860-8c680e97-2f03-487e-a860-abf7709411df.png">

(sdist is identical except for this `setup.cfg` file)

# Docs

Explicit guidance from PyPA to remove it (or set to zero) when dropping Python 2:

> Traditionally, projects providing Python code that is semantically compatible with both Python 2 and Python 3, produce [wheels](https://packaging.python.org/en/latest/glossary/#term-Wheel) that have a `py2.py3` tag in their names. When dropping support for Python 2, it is important not to forget to change this tag to just `py3`. It is often configured within `setup.cfg` under the `[bdist_wheel]` section by setting `universal = 1` if they use setuptools.

* https://packaging.python.org/en/latest/guides/dropping-older-python-versions/?highlight=universal#dealing-with-the-universal-wheels

Wheel docs:

* https://wheel.readthedocs.io/en/stable/user_guide.html?highlight=universal#building-wheels